packages/react-components/components/select/src/select.jsx

```diff
@@ -45,7 +50,7 @@ const renderAction = ({ content, key, className, ...rest }, index) => {
 };
 
 export function PureSelect(props) {
-    const { options, actions, inline, forwardedRef, ...rest } = props;
+    const { options, actions, inline, multiple, forwardedRef, ...rest } = props;
```

Comment:
You dont need to add `multiple` here since you don't use it, it would be provided to the select component through `...rest`.